### ISSUE TITLE
tagged text should not be escaped

### ISSUE THREAD

**Issue by GoogleCodeExporter** [txt2tags/txt2tags#11]
```
Txt2tags should never touch anything inside the ''tagged mark'', even the 
special escapes. Today, 
doFinalEscape() is performed inside tagged texts. Example in tex:

$ echo -e "\nTest ''pass/pass''" | txt2tags -i- -o- -H -t lout

@LP
Test pass"/"pass


-- thanks Barrie Stott for reporting
```

Original issue reported on code.google.com by `aureliojargas@gmail.com` on 7 Apr 2009 at 8:54

**Comment by fgallaire** [comment]
Don't understand all, but commit 20405d82494cd5789540ea86d72a182c2fde711 (formerly r895) introduce a bug for me, in the sample.t2t:
We use double *, /, - and _ to represent **bold**,
was converted in:
We use double *, /, - and \_ to represent \textbf{bold},
and now becomes:
We use double *, /, - and $\backslash$_ to represent \textbf{bold},
and doesn't compile correctly anymore.

**Comment by farvardin** [comment]
yes, I got this problem too on my LaTeX files, it's the reason why on
textallion I'm still using r804 of txt2tags (
https://bitbucket.org/farvardin/textallion/src/5010384ce0875d87cdde673b3d4344265f827e1a/contrib/txt2tags/txt2tags?at=default
)

You can find the diff of some errors I got there:
http://pastebin.com/xbbFCrWu

(t2t source file is at
https://bitbucket.org/farvardin/textallion/src/5010384ce0875d87cdde673b3d4344265f827e1a/docs/documentation_en.t2t?at=default
)

2015-07-04 3:30 GMT+02:00 Florent Gallaire <notifications@github.com>:

**Comment by jendrikseipp** [comment]
Thanks for the heads up! I could reproduce the error on my end and think it can be fixed by just swapping the position of the two Latex escapes. I am a little reluctant to do the fix though, because I don't see the point in fixing the same error in both txt2tags versions. Rather, I'd like to see us moving forward in issue #190.

**Comment by fgallaire** [comment]
@jendrikseipp txt2tags repo is now ready for your fix (for both txt2tags ^^).

**Comment by fgallaire** [comment]
@jendrikseipp could you apply your patch as this is the most important bug to fix before 2.7 release ?

**Comment by jendrikseipp** [comment]
This is now fixed in the v3 branch.
